test/util/service_context.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/codeready-toolchain/registration-service/pkg/kubeclient"
+	"github.com/codeready-toolchain/registration-service/pkg/server"
+	"github.com/codeready-toolchain/registration-service/test/fake"
+	commontest "github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewMemberClusterServiceContext(t *testing.T, cl client.Client) fake.MemberClusterServiceContext {
```

Comment:
Not related to your PR , but I'm a bit confused with the name of the function and the returning struct. 

Why does it refer to `Member` since it seems to contain a rest client for the host namespace and an application type ( which BTW it's again another rest client for the host operator namespace ) ?

Reply:
I have no clue why the type is named in this way :man_shrugging: 
I left it the same name to not incorporate even more changes and named the function accordingly. I hope that I'll be able to drop both of them soon.

Reply:
Ok makes sense. Thanks for answering and taking a look into it. I think ideally we would drop both indeed.